packages/knip/src/types/plugins.ts

```diff
@@ -33,7 +33,8 @@ export type ResolveEntryPaths<T = any> = (config: T, options: PluginOptions) =>
 // biome-ignore lint/suspicious/noExplicitAny: TODO
 export type ResolveConfig<T = any> = (config: T, options: PluginOptions) => Promise<string[]> | string[];
 
-export type Resolve = (options: PluginOptions) => Promise<string[]> | string[];
+// biome-ignore lint/suspicious/noExplicitAny: TODO
+export type Resolve<T = any> = (config: T, options: PluginOptions) => Promise<string[]> | string[];
```

Comment:
I added `config` as the first parameter to align the function's format with `ResolveEntryPaths` and `ResolveConfig`.